modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -41,11 +41,16 @@ You can use any of the following options for authentication when creating an API
 
 ** Using <<create_api-CRD, user credentials>> (username and password). 
 +
-This option supports only those users that are managed by Anypoint Platform.
+This option supports only those users that are:
+
+*** Managed by Anypoint Platform.
+*** Not configured with Multi-Factor authentication.
```

Comment:
say in a parallel fashion to the mfa statement above, so Have not enabled.... (or however you decide it).

Reply:
Done! Thanks!